cpp/src/interop/from_arrow.cu

```diff
@@ -304,7 +326,6 @@ std::unique_ptr<column> dispatch_to_cudf_column::operator()<cudf::string_view>(
                stream,
                mr);
 }
-
```

Comment:
*nit*: Probably didn't want to delete this line?

Reply:
Reverted it.